### ISSUE TITLE
step backwards (b) motion is not functioning properly

### ISSUE THREAD

**Issue by willdavidow** [chaoren/vim-wordmotion#25]
The motion to step backwards over/through words is no longer working after I recently updated the plugin. Previously it had stepped over camelCased words perfectly fine, but now the functionality (even default of stepping over entire words) is broken.

Now, When I press `b` the cursor moves backwards as if I were pressing the `h` key to step backwards letter by letter. 

I haven't looked at the code to try and figure out a possible PR, but I did verify that it's this plugin by removing the plugin, which returned the `b` key back to the standard behavior.

**Comment by chaoren** [comment]
I can't reproduce your problem.
I also haven't changed anything outside of documentation for a long time.
what's the output of `:map b`?
It should look like this:
```
o  b           * :<C-U>call <SNR>25_WordMotion(v:count1, 'o', 'b', [])<CR>
x  b           * :<C-U>call <SNR>25_WordMotion(v:count1, 'x', 'b', [])<CR>                                                          
n  b           * :<C-U>call <SNR>25_WordMotion(v:count1, 'n', 'b', [])<CR>
```

**Comment by chaoren** [comment]
What plugin manager did you use to install this plugin.

**Comment by willdavidow** [comment]
I used vim-plug to install the plugin.

So, this is weird - I literally changed nothing other than removing vim-wordmotion, using another plugin (CamelCaseMotion) for the evening yesterday, and then re-installing vim-wordmotion just now to verify the key mappings... and now the issue is gone. I've changed literally nothing else in my setup between then and now.

I guess it was somehow, something on my machine, or some sort of git hiccup? I have no idea... Apologies for the false alarm.
